x-pack/plugin/core/src/main/resources/monitoring-es.json

```diff
@@ -916,6 +916,83 @@
               }
             }
           }
+        },
+        "ccr_stats": {
+          "properties": {
+            "leader_index": {
+              "type": "keyword"
+            },
+            "shard_id": {
+              "type": "keyword"
```

Comment:
I am not sure about this being a keyword, I think probably a numeric field type is preferred here?

Reply:
Good point. It is serialized as a string now (`[index-name][shard-id]`). I think that is not super useful and storing just the shard id is better? (the leader index is already stored in another field)

Reply:
I think I am missing something, it should only be being serialized as the numerical shard ID, without the index name. See this test:

https://github.com/elastic/elasticsearch/blob/23f12e42c13878b0ff3b4c33420eb1570ee586cd/x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/collector/ccr/CcrStatsMonitoringDocTests.java#L142

and the x-content logic:

https://github.com/elastic/elasticsearch/blob/23f12e42c13878b0ff3b4c33420eb1570ee586cd/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/ShardFollowNodeTaskStatus.java#L380

Reply:
whoops. I read `shardId` as field name and I assumed `ShardId` class.